### DISCUSSION TITLE
Request: Show the sender of a reaction

### DISCUSSION THREAD

**Discussion by nycki93** [stoatchat/discussions#583] [Q&A] [answered]
On Discord and Teams, reactions often serve the role of "non-verbal responses", those little nods and waves that let you acknowledge a message without replying to it. On Revolt, at least on the app and website, I don't see a way to see who sent a reaction. I suppose I could tell people to reply instead of react, but that's a bit clunky. Could we have a way to see who reacted, please?

**Comment by Rexogamer** [comment, marked as answer]
This is currently possible through the API - the new client we're working on (which should be ready and available somewhat soon) also supports this :3
